spec/controllers/custom_fields_controller_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'spec_helper'
+
+describe CustomFieldsController, type: :controller do
+  render_views
+
+  fixtures :projects, :users, :roles, :members, :member_roles, :issues, :issue_statuses, :versions,
+           :trackers, :projects_trackers, :issue_categories, :enabled_modules, :enumerations, :attachments,
+           :workflows, :custom_fields, :custom_values, :custom_fields_projects, :custom_fields_trackers,
+           :time_entries, :journals, :journal_details, :queries, :repositories, :changesets
+
+  include Redmine::I18n
+
+  before do
+    @controller = CustomFieldsController.new
+    @request = ActionDispatch::TestRequest.create
+    @response = ActionDispatch::TestResponse.new
+    User.current = nil
+    @request.session[:user_id] = 1 #permissions are hard
+  end
+
+  let(:project) { projects(:projects_001) }
+  let(:custom_field) { custom_fields(:custom_fields_009) }
+
+  describe "POST /" do
+    before { post :create, params: { custom_field: { name: "CustomField", type: "IssueCustomField", project_ids: [project.id] } } }
```

Comment:
This line seems to not load the `create` action at all.

Reply:
I have merged the changes. Thanks.
Please have a look at my last commits to see how I fixed that.